### PR TITLE
Complete editorial update

### PR DESCRIPTION
**Pull request by ricardoaerobr**
#### Types of changes
- [ ] New content (guides, endpoints, app documentation)
- [X] Improvement (make a documentation even better)
- [ ] Fix (fix a documentation error)
- [ ] Spelling and grammar accuracy (self-explanatory)

PR created to fix broken links and callout formats, and make editorial improvements to the article "Pix: Instant Payments in Brazil" (EDU-9714).

### CODE REVIEW

File: docs/guides/Integration-Guides/payments-integration-guide/payments-integration-pix-instant-payments-in-brazil.md [hunk 1]
```diff
@@ -8,10 +8,14 @@ updatedAt: "2022-02-03T15:19:06.694Z"
...
+Pix is available to both physical and legal persons, and both need to have an identifier key registered with some financial entity (banks, fintechs or payment institutions) to proceed with the transaction.
+
+According to the [eligibility criteria](https://www.bcb.gov.br/estabilidadefinanceira/participantespix) set forth by the BCB, certain financial entities will be required to offer this payment method, while others may optionally offer it or not be eligible to participate.
+
+In this article, we will explain how to extend your Payment Provider Protocol implementation to allow stores to offer Pix as an additional payment method to their clients.
 
-In this step, we’ll explain how to extend your Payment Provider Protocol implementation to allow stores to offer Pix as an additional payment method to their clients.
 ![These are some of the benefits of an instant payments ecosystem highlighted by the BCB](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/images/payments-integration-pix-instant-payments-in-brazil-0.png)
+
 [block:callout]
```
Comment (karenkrieger) [inline docs/guides/Integration-Guides/payments-integration-guide/payments-integration-pix-instant-payments-in-brazil.md:19]: Formatar callout

File: docs/guides/Integration-Guides/payments-integration-guide/payments-integration-pix-instant-payments-in-brazil.md [hunk 2]
```diff
@@ -46,13 +36,14 @@ If you are ready to develop the middleware that implements our Payment Provider
 
 - **The middleware must consistently respond within established response times.** We enforce a maximum response time of 5 seconds for homologation tests, as well as a maximum response time of 20 seconds to any other API request.
 
-While our protocol describes ten endpoints for implementation, not all of them are applicable when integrating Pix instant payments. Regarding the two provider flows:
+While our protocol describes 9 endpoints for implementation, not all of them are applicable when integrating Pix instant payments. Regarding the two provider flows:
+
+- [Payment Flow](https://developers.vtex.com/docs/api-reference/payment-provider-protocol#get-/manifest): 6 endpoints that must be **mandatory** implemented.
 
-- [Payment Flow](https://developers.vtex.com/vtex-rest-api/reference/payment-flow): its implementation is **mandatory**.
+- [Configuration Flow endpoints](https://developers.vtex.com/docs/api-reference/payment-provider-protocol#post-/authorization/token): 3 endpoints whose implementation is **optional and currently not available for Pix**.
```
Comment (karenkrieger) [inline docs/guides/Integration-Guides/payments-integration-guide/payments-integration-pix-instant-payments-in-brazil.md:43]:
```suggestion
- [Configuration Flow endpoints](https://developers.vtex.com/docs/api-reference/payment-provider-protocol#post-/authorization/token): three endpoints whose implementation is **optional and currently not available for Pix**.
```

File: docs/guides/Integration-Guides/payments-integration-guide/payments-integration-pix-instant-payments-in-brazil.md [hunk 3]
```diff
@@ -46,13 +36,14 @@ If you are ready to develop the middleware that implements our Payment Provider
...
 
-- [Configuration Flow](https://developers.vtex.com/vtex-rest-api/reference/configuration-flow): its implementation is **optional and currently not available for Pix**.
+The table below gives further detail on the applicability of each endpoint to Pix instant payments.
 
-The table below gives further detail on the applicability of each endpoint to Pix instant payments. More details on the integration steps are given in the rest of this tutorial.
 [block:parameters]
```
Comment (karenkrieger) [inline docs/guides/Integration-Guides/payments-integration-guide/payments-integration-pix-instant-payments-in-brazil.md:47]: Formatar código

File: docs/guides/Integration-Guides/payments-integration-guide/payments-integration-pix-instant-payments-in-brazil.md [hunk 4]
```diff
@@ -205,106 +295,107 @@ After the provider realizes the payment cancelation, we expect is a response lik
...
 }
-[/block]
-Check the complete endpoint documentation [here](https://developers.vtex.com/vtex-developer-docs/reference/inboundrequestbeta).
+```
 
-For more information and frequently asked questions, refer to [Pix - FAQ](https://help.vtex.com/pt/tutorial/pix-faq--3lx7zCU2lQroTEBCYKYbo3).
+>ℹ️ The Inbound Request (BETA) is mandatory only for integrations via Payment Provider Protocol with an external Payment App. If the Pix payment method was implemented via VTEX Payment App, the Inbound Request is not necessary. For more information, access the [Inbound Request (BETA)](https://developers.vtex.com/docs/api-reference/payment-provider-protocol#post-/payments/-paymentId-/inbound-request/-action-).
 
-## Wrapping up
+For more information about the Pix, access its [FAQ](https://help.vtex.com/pt/tutorial/pix-faq--3lx7zCU2lQroTEBCYKYbo3).
```
Comment (karenkrieger) [inline docs/guides/Integration-Guides/payments-integration-guide/payments-integration-pix-instant-payments-in-brazil.md:399]:
```suggestion
For more information about the Pix, access its [FAQ](https://help.vtex.com/en/tutorial/pix-faq--3lx7zCU2lQroTEBCYKYbo3).
```

**Review by karenkrieger — CHANGES_REQUESTED** [review]

File: docs/guides/Integration-Guides/payments-integration-guide/payments-integration-pix-instant-payments-in-brazil.md (hunk 4 above)
Comment (ricardoaerobr) [reply docs/guides/Integration-Guides/payments-integration-guide/payments-integration-pix-instant-payments-in-brazil.md]: Fixed.

File: docs/guides/Integration-Guides/payments-integration-guide/payments-integration-pix-instant-payments-in-brazil.md (hunk 2 above)
Comment (ricardoaerobr) [reply docs/guides/Integration-Guides/payments-integration-guide/payments-integration-pix-instant-payments-in-brazil.md]: Fixed.

File: docs/guides/Integration-Guides/payments-integration-guide/payments-integration-pix-instant-payments-in-brazil.md (hunk 1 above)
Comment (ricardoaerobr) [reply docs/guides/Integration-Guides/payments-integration-guide/payments-integration-pix-instant-payments-in-brazil.md]: Fixed.

File: docs/guides/Integration-Guides/payments-integration-guide/payments-integration-pix-instant-payments-in-brazil.md (hunk 3 above)
Comment (ricardoaerobr) [reply docs/guides/Integration-Guides/payments-integration-guide/payments-integration-pix-instant-payments-in-brazil.md]: Fixed.

**Review by karenkrieger — CHANGES_REQUESTED** [review]

**Review by karenkrieger — APPROVED** [review]
LGTM

**Pull request merged**